.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,75 @@
+name: s3rver
+
+on:
+  push:
+    branches:
+      - master
+      - next
+  pull_request:
+    branches:
+      - master
+      - next
```

Comment:
Any purpose for limiting branches here? Are there certain branches we don't want this to run on, and why?

Reply:
The branch limitation avoids double building in case of pushing and immediately creates a pull request by the project developer. However, for this purpose it is enough to limit the branches for pushs.